src/main/scala/devices/tilelink/Plic.scala

```diff
@@ -135,71 +136,82 @@ class TLPLIC(params: PLICParams, beatBytes: Int)(implicit p: Parameters) extends
     require(nHarts > 0 && nHarts <= PLICConsts.maxHarts, s"Must be: nHarts=$nHarts > 0 && nHarts <= PLICConsts.maxHarts=${PLICConsts.maxHarts}")
 
     // For now, use LevelGateways for all TL2 interrupts
-    val gateways = Vec((false.B +: interrupts).map { case i =>
+    val gateways = interrupts.map { case i =>
```

Comment:
how are you able to get rid of the 0th gateway?

Reply:
The 0th gateway was always a no-op. The 0 interrupt never fires.